app/views/search/_filter.html.erb

```diff
@@ -1,17 +1,17 @@
 <% return if values.blank? %>
 
 <div class="category">
-  <button class="filter-label <%= 'expanded' if @enhanced_query[category.to_sym].present? || first == true %>"
+  <button class="filter-label <%= 'expanded' if @enhanced_query[category].present? || first == true %>"
```

Comment:
I see us dropping the `.to_sym` method here, and I'm not sure why. I'm sure it makes sense - my first assumption is because the values aren't convertible to symbols now, but were before. I'm just not sure I'm following what changed.

Reply:
Sorry, I should've been clearer in the commit message. That `to_sym` call was there because `SearchController#extract_filters` didn't always cast the filter keys as symbols. I refactored that method to do so a while back, but noticed while debugging this that there were still a couple of `to_sym` calls. I removed them to make this less confusing for our future selves (hopefully), but it's unrelated to the additional/customizable filters changeset.

Reply:
Ah, gotcha - that makes sense. Thanks!